internal/repository/sync.go

```diff
@@ -23,9 +26,10 @@ var (
 
 // Clone wraps a typical git clone invocation, fetching gittuf refs in addition
 // to the standard refs. It performs a verification of the RSL against the
-// specified HEAD after cloning the repository.
+// specified HEAD after cloning the repository. If called with the expected
+// root keys, it will verify that the cloned root keys match the expected ones.
 // TODO: resolve how root keys are trusted / bootstrapped.
```

Comment:
I think the todo can be taken out if the root keys are specified explicitly.

Reply:
Fixed